.github/workflows/inappmessaging.yml

```diff
@@ -38,9 +38,13 @@ jobs:
     - name: Setup Bundler
       run: scripts/setup_bundler.sh
     - name: Prereqs
-      run:  scripts/install_prereqs.sh InAppMessaging ${{ matrix.platform }} xcodebuild
+      run: |
+       scripts/install_prereqs.sh InAppMessaging ${{ matrix.platform }} xcodebuild
+       scripts/install_prereqs.sh InAppMessagingSwift ${{ matrix.platform }} xcodebuild
     - name: Build and test
-      run:  scripts/third_party/travis/retry.sh scripts/build.sh InAppMessaging ${{ matrix.platform }} xcodebuild
+      run: |
+       scripts/third_party/travis/retry.sh scripts/build.sh InAppMessaging ${{ matrix.platform }} xcodebuild
```

Comment:
Putting the tests in separate jobs either via matrix or just another block makes it easier to debug.